database/types.go

```diff
@@ -190,6 +190,10 @@ type BlockBuilderEntry struct {
 
 	IsHighPrio    bool `db:"is_high_prio"   json:"is_high_prio"`
 	IsBlacklisted bool `db:"is_blacklisted" json:"is_blacklisted"`
+	IsDemoted     bool `db:"is_demoted" json:"is_demoted"`
```

Comment:
nit: should there be more space after `"is_demoted"` so that `json:` aligns?

Reply:
yup, done!